code/modules/ruins/spaceruin_code/hilbertshotel.dm

```diff
@@ -0,0 +1,345 @@
+    Goodbye<br>
+	<br>
+	<i>Doctor Hilbert</i>"}
+
+/obj/item/paper/crumpled/robertsworkjournal
+    name = "Work Journal"
+    info = {"<h4>First Week!</h4>
+	First week on the new job. It's a secretarial position, but hey, whatever pays the bills. Plus it seems like some intersting stuff goes on here.<br>
+	Doc says its best that I don't openly talk about his research with others, I guess he doesn't want it getting out or something. I've caught myself slipping a few times when talking to others, it's hard not to brag about someting this cool!<br>
+	I'm not really sure why I'm choosing to journal this. Doc seems to log everything. He says it's incase he discovers anything important.<br>
+    I guess thats why I'm doing it too, I've always wanted to be a part of something important.<br>
+    Here's to a new job and to becoming a part of something important!<br>
+	<h4>Weird times...</h4>
+	Things are starting to get a little strange around here. Just weeks after Doc's amazing breakthrough, weird visitors have began showing up unannouced, asking strange things about Doc's work.<br>
+    I knew Doc wasn't a big fan of company, but even he seemed strangely unnerved when I told him about the visitors.<br>
+    He said it's important that from here on out we keep tight security on everthing, even other staff members.<br>
+    He also said something about securing data, something about sixty-four bases. What's that mean? Something to do with baseball? Doc never struct me as the sports type...<br>
+    He often uses a lot of big sciencey words that I don't really understand, but I kinda dig it, it makes me feel like I'm witnessing something big.<br>
+    I hope things go back to normal soon, but I guess that's the price you pay for being a part of something important.<br>
+	<h4>Last day I guess?</h4>
+	Thinks are officially starting to get too strange for me.<br>
+    The visitors have been coming a lot more often, and they all seem increasingly aggressive and nosey. I'm starting to see why they made Doc so nervous, they're certainly starting to creep me out too.<br>
+    Awhile ago Doc started having me keep the place on strict lockdown and requested I refuse entry to anyone else, including previous staff.<br>
+    But the weirdest part?<br>
+    I haven't seen Doc in days. It's not unusual for him to work continuosly for long periods of time in the lab, but when I took a peak in their yesterday - he was nowhere to be seen! I didn't risk prying much further, Doc had a habit of leaving the defense systems on these last few weeks.<br>
+    I'm thinking it might be time to call it quits. Can't work much without a boss, plus things are starting to get kind of shady. I wanted to be a part of something important, but you gotta know when to play it safe.<br>
+    As my dad always said, "The smart get famous, but the wise survive..."<br>
+	<br>
+	<i>Robert P.</i>"}
+
```

Comment:
There are numerous typos scattered in these journal logs, I'd highly recommend you go through with a word editor and spellcheck everything if you weren't planning to already.